cmd/kiam-check/README.md

```diff
@@ -0,0 +1,58 @@
+## KIAM Check
+
+This check tests `KIAM` servers and agents running within your cluster can properly intercept AWS metadata service requests. This check utilizes a `KIAM` annotation for querying Lambdas which can be set via your `KuberhealthyCheck` custom resource by passing in a field under `spec` like so:
```

Comment:
maybe be explicit with the kind of arn needed for this check and why we're querying lambdas?